src/types.ts

```diff
@@ -29,6 +29,10 @@ export interface KBarOptions {
     enterMs?: number;
     exitMs?: number;
   };
+  hotkeys?: {
+    metaKey?: boolean;
+    ctrlKey?: boolean;
+  };
```

Comment:
I think `hotkeys` here is too general – are we talking about hotkeys for triggering the command bar, shortcuts, or in this case the up/down navigation controls?

Reply:
@timc1 this is my case:
https://github.com/timc1/kbar/issues/148